### PR TITLE
Wireframe testing

### PR DESCRIPTION
**Pull request by ola-show**
testing

### CODE REVIEW

**Comment by ola-show** [comment]
I need someone to please review my work.

**Comment by webxacademy** [comment]
great job Ola!

**Comment by webxacademy** [comment]
> I need someone to please review my work.

i am on it

**Comment by ola-show** [comment]
> > I need someone to please review my work.
> 
> i am on it

oh okay thanks,

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # lotus_salon
 Lotus Salon Project
 
+test!
```
Comment (webxacademy) [inline README.md:4]: looks okay, but seems like you didn't do much

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
 
+tesing testing
```
Comment (webxacademy) [inline README.md:6]: I see you added testing comments only.

File: README.md (hunk 1 above)
Comment (ola-show) [reply README.md:4]: yes that was all that was required of me to test it out.

**Review by webxacademy — CHANGES_REQUESTED** [review]
Can you reduce the text to just one instead.

**Pull request closed without merging**